definitions/reports/rbac.rb

```diff
@@ -0,0 +1,61 @@
+module Checks
+  module Report
+    class RBAC < ForemanMaintain::Report
+      metadata do
+        description 'Checks the RBAC use'
+      end
+
+      # How many users do you have in the system?
+      # How many non-admin users do you have?
+      # How many custom roles did you create and assigned to users?
+      # rubocop:disable Layout/LineLength
+      # rubocop:disable Metrics/MethodLength
+      # rubocop:disable Metrics/AbcSize
```

Comment:
Break this method up instead of disabling rubocop. This could also help with the comments listed in these files. You can use ruby to help, e.g. 

```
def number_of_users
def number_of_non_admin_users
def number_of_custom_roles
```